src/player.h

```diff
@@ -225,7 +226,7 @@ class player_t : public game_object_t {
   void decrease_serf_count(int type) { serf_count[type]--; }
   int *get_serfs() { return reinterpret_cast<int*>(serf_count); }
 
```

Comment:
du you know what the serf_count[] is and why increase_res_count() is increasing this as well as decrease_serf_count() did?